phpcs.xml

```diff
@@ -2,6 +2,10 @@
 <ruleset name="Zend Framework Coding Standard">
     <rule ref="./vendor/zendframework/zend-coding-standard/ruleset.xml"/>
 
+    <rule ref="ZendCodingStandard.Commenting.FunctionComment.TypeHintMissing">
+        <severity>0</severity>
+    </rule>
```

Comment:
We disable this rule for the whole library right now. The issue is only in [`Emitter/EmitterStack.php`](https://github.com/zendframework/zend-expressive/blob/master/src/Emitter/EmitterStack.php) lines [`58`](https://github.com/zendframework/zend-expressive/blob/master/src/Emitter/EmitterStack.php#L54-L58), [`71`](https://github.com/zendframework/zend-expressive/blob/master/src/Emitter/EmitterStack.php#L67-L71), [`84`](https://github.com/zendframework/zend-expressive/blob/master/src/Emitter/EmitterStack.php#L80-L84). In PHPDocs we have said parameter `$emitter` should have type `EmitterInterface` but there is missing type hint.

Hm... Now I think maybe better will be just exclude only this file (`EmitterStack`) from this rule instead of disabling the rule for the whole library?